lotto.css

```
@@ -0,0 +1,180 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+html, body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    font-weight: 400;
+    font-family: Roboto;
+    font-size: 15px;
+    line-height: 24px;
+    min-height: 855px;
+    min-width: 414px;
+}
+
+/* 헤더,푸터 */
+header{
+    position: fixed;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    background-color: #4E5BA6;
+    height: 64px;
+    top: 0;
+}
+
+header p{
+    position: absolute;
+    color: white;
+    left: 130px;
+}
+
+
+footer{
+    width: 100%;
+    position: fixed;
+    bottom: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 80px;
+    border: 1px solid;  
```

위에만 주어도 될 것 같아요! 그리고 색도 넣어주면 더욱 좋겠네요!

```suggestion
    border-top: 1px solid #4E5BA633;  
```